ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -0,0 +1,400 @@
+//! The Yk JIT IR
+
+use std::fmt;
+use strum_macros::FromRepr;
+
+/// Number of bits used to encode an opcode.
+const OPCODE_SIZE: u64 = 8;
```

Comment:
I think this stuff should be gated behind a `#[cfg(x86)]` thing or similar. We could even gate the entire module for the time being.

Reply:
@vext01 tells me that the intention is for this to be a 64-bit layout regardless of the platform. We should add a doc string to the module to that effect, just so that nervous people like me know.

Reply:
This is just the number of bits used to encode an opcode. It isn't x86_64 specific.

Reply:
Documented in 731d1ae92873424f75dbb0244578b47ee7929872